src/main/java/step3/model/machine/LottoManualGenerator.java

```diff
@@ -1,38 +1,27 @@
 package step3.model.machine;
 
 import java.util.Arrays;
-import java.util.HashSet;
 import java.util.List;
 import java.util.stream.Collectors;
+import step3.model.lotto.LottoNumber;
 import step3.model.util.InputValidation;
-import step3.model.value.ErrMsg;
 import step3.model.value.Rule;
 
 public class LottoManualGenerator {
-    public List<Integer> createLotto(String lottoManualInput) {
+    public List<LottoNumber> createLotto(String lottoManualInput) {
         InputValidation.validateEmpty(lottoManualInput);
-        List<Integer> lottoNumbers = splitLottoNumbers(lottoManualInput);
-        verifyLotto(lottoNumbers);
-        return lottoNumbers;
+        return splitLottoNumbers(lottoManualInput);
     }
 
-    private void verifyLotto(List<Integer> lottoNumbers) {
-        if(lottoNumbers.size() != Rule.LOTTO_NUMBER_LENGTH){
-            throw new IllegalArgumentException(ErrMsg.WRONG_LENGTH);
-        }
-        if(lottoNumbers.size() != new HashSet<>(lottoNumbers).size()){
-            throw new IllegalArgumentException(ErrMsg.DUPLICATED_INPUT);
-        }
-    }
-
-    private List<Integer> splitLottoNumbers(String lottoManualInput) {
+    private List<LottoNumber> splitLottoNumbers(String lottoManualInput) {
         return Arrays.stream(lottoManualInput.split(Rule.DELIMITER))
-                .map(this::changeToInt).collect(Collectors.toList());
+                .map(this::changeToInt).map(LottoNumber::new).collect(Collectors.toList());
```

Comment:
```suggestion
                .map(this::changeToInt)
                .map(LottoNumber::new)
                .collect(Collectors.toList());
```
객체지향 생활 체조 원칙에서는 한 줄에 점을 하나씩만 찍는것을 원칙으로 하고있어요!😀

[객체지향 생활 체조 원칙](https://blogshine.tistory.com/241)